docs/tutorials/first-run-experience-tutorial.md

```diff
@@ -0,0 +1,152 @@
+    - [OneNote](../quickstarts/onenote-quickstart.md)
+    - [Outlook](../quickstarts/outlook-quickstart.md)
+    - [PowerPoint](../quickstarts/powerpoint-quickstart.md)
+    - [Project](../quickstarts/project-quickstart.md)
+    - [Word](../quickstarts/word-quickstart.md)
+
+1. Follow the instructions in your selected quick start. After you complete its "Try it out" section, return here to continue.
+
+## Implement the first-run experience
+
+### Update the HTML file
+
+Be clear about the area of the UI that will be part of the first-run experience. In this tutorial, you'll create a div with `id` named "first-run-experience" that represents what users see only the first time they run your add-in.
+
+1. Open the **taskpane.html**. Replace the `<main>` element with the following markup, then save the file. Note about this markup:
+
+    - The "first-run-experience" div is inserted in the `<main>` element. It surrounds the list of Office Add-ins features. By default, this div isn't displayed.
+    - The first `<p>` element provides the user with instructions for using the add-in.
+
+    ```html
+    <main id="app-body" class="ms-welcome__main" style="display: none;">
+        <div id="first-run-experience" style="display: none;">
+            <h2 class="ms-font-xl"> Discover what Office Add-ins can do for you today! </h2>
+            <ul class="ms-List ms-welcome__features">
+                <li class="ms-ListItem">
+                    <i class="ms-Icon ms-Icon--Ribbon ms-font-xl"></i>
+                    <span class="ms-font-m">Achieve more with Office integration</span>
+                </li>
+                <li class="ms-ListItem">
+                    <i class="ms-Icon ms-Icon--Unlock ms-font-xl"></i>
+                    <span class="ms-font-m">Unlock features and functionality</span>
+                </li>
+                <li class="ms-ListItem">
+                    <i class="ms-Icon ms-Icon--Design ms-font-xl"></i>
+                    <span class="ms-font-m">Create and visualize like a pro</span>
+                </li>
+            </ul>
+        </div>
+        <p class="ms-font-l">Select any range of cells in the worksheet, then click <b>Run</b>.</p>
+        <div role="button" id="run" class="ms-welcome__action ms-Button ms-Button--hero ms-font-xl">
+            <span class="ms-Button-label">Run</span>
+        </div>
+        <p><label id="item-subject"></label></p>    
+    </main>
+    ```
+
+1. If you selected an Office application besides Excel, update the first `<p>` element with more appropriate instructions.
+
+### Update the JavaScript file
+
+Update the JavaScript file to display the first-run experience if this is the first time the user is running the add-in.
+
+1. Open the **taskpane.js** file. Replace the `Office.onReady` statement with the following code, then save the file. Note about this code:
+
+    - It checks local storage for a key called "showedFRE". If it doesn't exist, then show the first-run experience.
```

Comment:
Since "it" refers to the add-in in the previous sentence, consider clarifying the noun in the second sentence.
```suggestion
    - It checks local storage for a key called "showedFRE". If the key doesn't exist, then show the first-run experience.
```